api/v1alpha1/virtualmachine_types.go

```diff
@@ -106,6 +106,10 @@ const (
 	// marked userConfigurable and already present in either OVF Properties of a VirtualMachineImage
 	// or as vApp properties on an existing VM or VMTX will be set, all others will be ignored.
 	VirtualMachineMetadataOvfEnvTransport VirtualMachineMetadataTransport = "OvfEnv"
+
+	// VirtualMachineMetadataCloudInitPrepTransport will set the VirtualMachineMetadata ConfigMap data as
```

Comment:
This wording is _super_ confusing. Can we add a clarification here that it is explicitly _not_ **cloud-init** metadata.

Reply:
Suggest:

```golang
// VirtualMachineMetadataCloudInitTransport indicates the data in
// the ConfigMap resource specified in
// VirtualMachineMetadata.ConfigMapName is cloud-init userdata.
//
// Please note, despite the name, that VirtualMachineMetadata has no
// relationship to cloud-init instance metadata. 
//
// For more information on cloud-init user data, please see
// https://cloudinit.readthedocs.io/en/latest/topics/format.html.
```

Reply:
I'll add this comment. This clarifies the confusion that could arise because of the name.